kpm/lib/kpm/system_helpers/cpu_information.rb

```diff
@@ -0,0 +1,70 @@
+module SystemProxy
```

Comment:
Could you namespace all modules under `KPM` (e.g. `KPM::SystemProxy`)? This is to prevent any leakage when KPM is used as a library (FYI, we've started developing an [Ansible module](https://github.com/killbill/killbill-cloud/blob/a926c1bf95319f9d7240dcda0d0e0226048bf1d5/ansible/library/killbill_facts#L30) using the gem).